.github/workflows/pull-request.yml

```
@@ -245,27 +245,51 @@
           npm run test:vitest:coverage
 
-      - name: Merge Coverage Reports
-        if: steps.changed-files.outputs.only_changed != 'true'
+      - name: Upload Vitest Coverage to Codecov
+        if: steps.changed-files.outputs.any_changed == 'true'
+        uses: codecov/codecov-action@v4
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          files: ./coverage/vitest/lcov.info
+          flags: vitest
+          gcov_ignore: 'docs/'
```

actionlint (1.7.4)

input "gcov_ignore" is not defined in action "codecov/codecov-action@v4". available inputs are "codecov_yml_path", "commit_parent", "directory", "disable_file_fixes", "disable_safe_directory", "disable_search", "dry_run", "env_vars", "exclude", "fail_ci_if_error", "file", "files", "flags", "git_service", "handle_no_reports_found", "job_code", "name", "network_filter", "network_prefix", "os", "override_branch", "override_build", "override_build_url", "override_commit", "override_pr", "plugin", "plugins", "report_code", "root_dir", "slug", "token", "url", "use_legacy_upload_endpoint", "use_oidc", "verbose", "version", "working-directory"

(action)